index.js

```diff
@@ -1 +1,50 @@
-console.log("Code your solution!")
+let todoForm = document.querySelector('#todo-item');
+
+todoForm.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    let userList = document.querySelector('ul');
+    let userInput = event.target['list-item'].value;
+    if (!userInput) {
+        let p = document.createElement('p');
+        p.textContent = 'Error. Todo cannot be empty';
+        todoForm.after(p);
+    } else {
+        let listItem = document.createElement('li');
+        listItem.textContent = userInput;
+        userList.append(listItem);
+        
+        // let deleteButton = document.createElement('button');
+        // deleteButton.setAttribute('type', 'submit');
+        // deleteButton.textContent = 'Delete';
+        // listItem.append(deleteButton);
+        event.target['list-item'].value = "";
+        // deleteButton.addEventListener('click', () => {
+        //     listItem.remove();
+        // });
+    }
+
+    let todoItems = document.querySelectorAll('li');
+
+    // for (let li of todoItems) {
+    //     console.log(li.style.textDecoration)
+    //     li.addEventListener('click', () => {
+    //         li.style.textDecoration = 'line-through';
+    //         // console.log(li.style.textDecoration)
+    //     });
+    // }
+    
+    // for (let li of todoItems){
+    //     if (li.style.textDecoration === 'line-through') {
+    //         li.addEventListener('click', () => {
+    //             li.style.textDecoration = 'none';
+    //         });
+    //     }
+    // }
+
+    for (let li of todoItems) {
+        li.addEventListener('click', () => {
+            li.style.textDecoration = 'line-through';
+            });
+        }
```

Comment:
```suggestion
listItem.addEventListener('click', () => {
   listItem.style.textDecoration = 'line-through'
   }
```
You don't have to loop through all of the `li`s to add this event listener each time you create a new one, you can just add the event listener to each single `li` at the time of its creation. `listItem` is the variable that you used to save the new `li` element on line 13.

Reply:
Thanks for the tip, Sam!